### PR TITLE
Event create_jobs use ActiveJob

### PR DESCRIPTION
**Pull request by evanrolfe**
Part of this card: https://trello.com/c/Tx9d3bUb/467-5-p4-event-system-change-delayed-jobs-to-be-created-with-activejobs

### CODE REVIEW

File: src/api/app/models/event/base.rb [hunk 1]
```diff
@@ -151,13 +151,13 @@ def perform_create_jobs
...
       Rails.logger.debug "PCJ #{inspect} #{create_jobs.inspect}"
       create_jobs.each do |job|
-        eclass = job.to_s.camelize.safe_constantize
-        raise "#{job.to_s.camelize} does not map to a constant" if eclass.nil?
-        djob = eclass.new(self)
-        raise("#{job.to_s.camelize} is not a CreateJob") unless djob.is_a?(CreateJob)
-        opts = {}
-        opts = { queue: eclass.job_queue } if eclass.methods.include? :job_queue
-        Delayed::Job.enqueue djob, opts
+        job_class = job.to_s.camelize.safe_constantize
```
Comment (hennevogel) [inline src/api/app/models/event/base.rb]: Why don't you just remove this? :-)

Comment (evanrolfe) [inline src/api/app/models/event/base.rb]: Will still need this method to schedule the jobs because we still rely on event.undone_jobs column.

Comment (hennevogel) [inline src/api/app/models/event/base.rb]: You can still inherit from `CreateJob`, I don't mind that. But there is no reason to schedule them with this method, is there?

Comment (evanrolfe) [inline src/api/app/models/event/base.rb]: So where should I schedule the jobs?

Comment (hennevogel) [inline src/api/app/models/event/base.rb]: We've talked about this and agreed that we kill `create_job` in another PR

**Pull request merged**